integration_test/testdata/test-foo.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
```

Comment:
🤔 can we symlink these from existing testdata dir?

Reply:
So I looked at the existing test-data we have and I could find nothing with valid yaml+archive. I think it would be possible without having these testdata in the repo, but the cost will be a more complex test setup. So my current gut feeling is that these new testdata will cause the smallest pain in the future.

Reply:
You're right, it seems like all testdata dirs are unit-test oriented and incomplete.

I think you can create a very simple "foo" plugin in `integration_test/testdata` here with a tarballed bash script, and a manifest for it, and just use it to test.

Reply:
Yes, I called it `test-foo`, but I can rename to `foo` as well.